lutris/xdgmenu.py

```diff
@@ -1,30 +0,0 @@
-"""Get games available from the XDG menu."""
```

Comment:
Was this file actually used anywhere? I don't see any removals of or updates for references to this.